### PR TITLE
feat: add functions lab

### LINKED ISSUE #431: chore: add lab for functions in frontend cert(JS only)
The previous workshop and lecture videos will teach campers about functions.

So we need a lab for them to do immediately after that.

- let
- const
- assignment
- reassignment
- basic string usage
- console.log
- camelCase
- string concatenation using the + operator
- template literals
- str.length property
- bracket notation and retrieving characters from strings
- str.indexOf method
- Number()
- Math.floor
- Math.random
- getting a random number from a range
- prompt()
- ===
- if/else if/else
- true and false (truthy and falsey)
- comparison operators(< >=)
- logical operators (|| &&)
- array basics
- push
- pop
- shift
- unshift
- basic object usage
- for loop
- for of loop
- do/while loop
- while loop
- forEach loop
- regular functions
- arrow functions

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/js-projects` directory
- [ ] add your completed project in an `script.js` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by ihechikara**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #431

### CODE REVIEW

File: frontend-cert/js-projects/functions/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+2. You should create a function named `subtract`. It should return the difference of two parameters.
+
+3. You should create an arrow function named `multiply`. It should return the product of two parameters.
+
+4. You should create an arrow function named `divide`. It should return the quotient of two parameters. The function should:
+    - Check if the second parameter is zero.
+    - Return an error message if the second parameter is zero.
+    - Return the quotient of the two parameters if the second parameter is not zero.
+
+5. You should create a function named `calculator` function. In the function's body:
```
Comment (zairahira) [inline frontend-cert/js-projects/functions/user-stories.md:12]:
```suggestion
5. You should create a function named `calculator`. In the function's body:
```

**Review by zairahira — COMMENTED** [review]
It's looking good @ihechikara 🎉

**Comment by jdwilkin4** [comment]
Hey @ihechikara 

We will actually have a workshop right before this that has campers build a calculator.
So we will need to come up with another project idea here.

**Comment by ihechikara** [comment]
> Hey @ihechikara
> 
> We will actually have a workshop right before this that has campers build a calculator. So we will need to come up with another project idea here.

Thanks for pointing that out, @jdwilkin4. I've replaced it with a random password generator project.

**Comment by zairahira** [comment]
The project is looking good, no changes from my end.

**Review by moT01 — APPROVED** [review]
LGTM @ihechikara 🎉

**Pull request merged**